pkg/proxy/iptables/proxier.go

```diff
@@ -1099,6 +1126,52 @@ func (proxier *Proxier) syncProxyRules() {
 			activeNATChains[svcXlbChain] = true
 		}
 
+		if proxier.localDetector.IsImplemented() && svcChain != "" {
+			// We have a LocalDetector, so fill in the rules in the SVC chain to
+			// redirect external traffic (or simply masquerade it if we don't
+			// have a separate external chain).
+			var externalTrafficDest utiliptables.Chain
```

Comment:
I think it would be easier to read this block without the variable.

```
args = [
    "-A", string(svcChain),
    "-m", "comment", "--comment", svcNameString,
    "-m", "addrtype",
    "!", "--src-type", "LOCAL"
]
if svcXlbChain != "" {
    proxier.natRules.Write(args, proxier.localDetector.JumpIfNotLocal(nil, svcXlbChain)
}else {
    proxier.natRules.Write(args, proxier.localDetector.JumpIfNotLocal(nil, KubeMarkMasqChain)
}
```

Reading the args first gives context.  Even if you just put the args before the variable and then Write() at the end.

Or maybe just calling the variable "jumpTo" or something less overlapping with `svcXlbChain`.  It's a pretty localized variable scope, so a fancy name it not super needed.